### PR TITLE
Convert some classes to ES6

### PR DESCRIPTION
**Pull request by adrianheine**
Needs `buble@0.15.1`, but `buble-rollup-plugin@0.15.0` can pull that in.

### CODE REVIEW

File: src/display/scrollbars.js [hunk 1]
```diff
@@ -27,26 +27,26 @@ export function measureForScrollbars(cm) {
...
+    on(horiz, "scroll", () => {
+      if (horiz.clientWidth) scroll(horiz.scrollLeft, "horizontal")
+    })
+
+    this.checkedZeroWidth = false
+    // Need to set a minimum width to see the scrollbar on IE7 (but must not set it on IE8).
+    if (ie && ie_version < 8) this.horiz.style.minHeight = this.vert.style.minWidth = "18px"
+  }
 
-NativeScrollbars.prototype = copyObj({
```
Comment (marijnh) [inline src/display/scrollbars.js]: Why did you remove the copyObj? This'll remove the `constructor` property from instances.

**Comment by marijnh** [comment]
I merged your Bublé patch. Not sure when the next release will happen.

File: src/display/scrollbars.js (hunk 1 above)
Comment (marijnh) [inline src/display/scrollbars.js]: Err, duh, you made it a regular class, so that's not an issue at all. Ignore this comment.

**Comment by adrianheine** [comment]
That Bublé patch is not necessary for this, since there are no object literals with short-hand methods, only classes.

**Comment by marijnh** [comment]
You don't appear to be using any inheritance or class expressions, so it should work with 0.15.0 too, no?

In any case, I've merged this.
